src/components/index.ts

```diff
@@ -1,2 +1,3 @@
 export { Button, ButtonProps } from "./button";
-export { Select, SelectProps} from "./select";
+export { Select, SelectProps } from "./select";
+export { Input, InputProps } from "./input";
```

Comment:
I think theres a slightly more intuitive way to export and import these things ( that doesnt rely just on memorizing names), but want to get everyones thoughts on this, and thats to mirror the directory structure, so you have an obvious convention that is self documenting and avoids naming collisions.

Example:
```js
// in index.ts
export * as input from './input'

// in input/index.ts
export * from './Input'

// usage

import {input} from 'uma-react';
const {Props:InputProps, Component:Input} = input;
```
This kind of implies that each component follows the pattern of exporting a props type and a "Component" object. 

This might get a bit tricky when you have multiple types of the same component, like if you had different types of buttons, but heres a suggestion:

```js
import {buttons} from 'uma-react'
const PrimaryButton = buttons.primary.Component;
const OutlineButton = buttons.outline.Component;
```

I'm not super opinionated but i just want to make it super easy to intuit how to both export components when building and import components when consuming

Reply:
Yeah I'd rather just import a component directly. Primary or Secondary or (whatever) should be in a theme prop or something.